torch/quantization/observer.py

```diff
@@ -586,8 +586,8 @@ def __init__(self, ch_axis=0, dtype=torch.quint8,
                                                        quant_min=quant_min,
                                                        quant_max=quant_max)
         self.ch_axis = ch_axis
-        self.register_buffer('min_vals', torch.tensor([]))
-        self.register_buffer('max_vals', torch.tensor([]))
+        self.min_vals = Parameter(torch.tensor([]), requires_grad=False)
```

Comment:
This has implications on distributed training. When these are buffers, all replicas will use the same values as the master (broadcast), which is needed during training, so that all replicas use the same scale and zero-point for fake-quant. When they are parameters, they are not synchronized, which can cause an accuracy drop.

Reply:
I see, is this true even if we set `requires_grad=False` on the parameters?